packages/perspective-bench/bench/versions.js

```diff
@@ -34,7 +34,7 @@ const JPMC_VERSIONS = [
 
 const FINOS_VERSIONS = ["0.3.1", "0.3.0", "0.3.0-rc.3", "0.3.0-rc.2", "0.3.0-rc.1"];
 
-const UMD_VERSIONS = ["0.4.5", "0.4.4", "0.4.2", "0.4.1", "0.4.0", "0.3.9", "0.3.8", "0.3.7", "0.3.6"];
+const UMD_VERSIONS = ["0.4.8", "0.4.7", "0.4.6", "0.4.5", "0.4.4", "0.4.2", "0.4.1", "0.4.0", "0.3.9", "0.3.8", "0.3.7", "0.3.6"];
```

Comment:
Thanks for updating this - reminds me, we could just use this and dump the pre-finos versions I think now?

```
$ npm view @finos/perspective  versions  --json
[
  "0.3.0-rc.1",
  "0.3.0-rc.2",
  "0.3.0-rc.3",
  "0.3.0-rc.4",
  "0.3.0",
  "0.3.1",
  "0.3.4",
  "0.3.5",
  "0.3.6",
  "0.3.7",
  "0.3.8",
  "0.3.9",
  "0.4.0-rc.1",
  "0.4.0-rc.2",
  "0.4.0-rc.3",
  "0.4.0-rc.4",
  "0.4.0-rc.5",
  "0.4.0-rc.6",
  "0.4.0",
  "0.4.1",
  "0.4.2",
  "0.4.4",
  "0.4.5",
  "0.4.6",
  "0.4.7",
  "0.4.8"
]
```